testing/file_test/file_test_base.cpp

```diff
@@ -147,9 +156,11 @@ auto FileTestBase::ProcessTestFileAndRun(TestContext& context)
 
   // Pass arguments as StringRef.
   llvm::SmallVector<llvm::StringRef> test_args_ref;
+  llvm::SmallVector<const char*> test_argv_ref;
   test_args_ref.reserve(context.test_args.size());
```

Comment:
Please either also reserve for your new list, or remove this for symmetry (it's a test so either way seems fine)

Reply:
Done.